code/game/objects/items/weapons/holy_weapons.dm

```diff
@@ -0,0 +1,328 @@
+	desc = "This hand of yours glows with an awesome power!"
+	flags = ABSTRACT | NODROP
+	w_class = 5
+	hitsound = 'sound/weapons/sear.ogg'
+	damtype = BURN
+	attack_verb = list("punched", "cross countered", "pummeled")
+
+/obj/item/weapon/nullrod/staff
+	icon_state = "godstaff-red"
+	item_state = "godstaff-red"
+	name = "red holy staff"
+	desc = "It has a mysterious, protective aura."
+	w_class = 5
+	force = 5
+	slot_flags = SLOT_BACK
+
+/obj/item/weapon/nullrod/staff/IsShield()
+	return 1
+
+/obj/item/weapon/nullrod/staff/blue
+	name = "blue holy staff"
+	icon_state = "godstaff-blue"
+	item_state = "godstaff-blue"
+
+/obj/item/weapon/nullrod/claymore
+	icon_state = "claymore"
+	item_state = "claymore"
+	name = "holy claymore"
+	desc = "A weapon fit for a crusade!"
+	w_class = 5
+	slot_flags = SLOT_BACK|SLOT_BELT
+	sharp = 1
+	edge = 1
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
+
+/obj/item/weapon/nullrod/claymore/IsShield()
+	if(prob(30))
+		return 1
+	else
+		return 0
+
+/obj/item/weapon/nullrod/claymore/darkblade
+	icon_state = "cultblade"
+	item_state = "cultblade"
+	name = "dark blade"
+	desc = "Spread the glory of the dark gods!"
+	slot_flags = SLOT_BELT
+	hitsound = 'sound/hallucinations/growl1.ogg'
+
+/obj/item/weapon/nullrod/claymore/chainsaw_sword
+	icon_state = "chainswordon"
+	item_state = "chainswordon"
+	name = "sacred chainsaw sword"
+	desc = "Suffer not a heretic to live."
+	slot_flags = SLOT_BELT
+	attack_verb = list("sawed", "torn", "cut", "chopped", "diced")
+	hitsound = 'sound/weapons/chainsaw.ogg'
+
+/obj/item/weapon/nullrod/claymore/glowing
+	icon_state = "swordon"
+	item_state = "swordon"
+	name = "force weapon"
+	desc = "The blade glows with the power of faith. Or possibly a battery."
+	slot_flags = SLOT_BELT
+
+/obj/item/weapon/nullrod/claymore/katana
+	name = "hanzo steel"
+	desc = "Capable of cutting clean through a holy claymore."
+	icon_state = "katana"
+	item_state = "katana"
+	slot_flags = SLOT_BELT | SLOT_BACK
+
+/obj/item/weapon/nullrod/claymore/saber
+	name = "light energy sword"
+	hitsound = 'sound/weapons/blade1.ogg'
+	icon_state = "swordblue"
+	item_state = "swordblue"
+	desc = "If you strike me down, I shall become more robust than you can possibly imagine."
+	slot_flags = SLOT_BELT
+
+/obj/item/weapon/nullrod/claymore/saber/red
+	name = "dark energy sword"
+	icon_state = "swordred"
+	item_state = "swordred"
+	desc = "Woefully ineffective when used on steep terrain."
+
+/obj/item/weapon/nullrod/sord
+	name = "\improper UNREAL SORD"
+	desc = "This thing is so unspeakably HOLY you are having a hard time even holding it."
+	icon_state = "sord"
+	item_state = "sord"
+	slot_flags = SLOT_BELT
+	force = 4.13
+	throwforce = 1
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
+
+/obj/item/weapon/nullrod/scythe
```

Comment:
with no force change this is literally better than stock null rod in basically every way